Scripts/Helpers/VRSmoothCamera.cs

```diff
@@ -78,12 +83,11 @@ void LateUpdate()
 			m_SmoothCamera.stereoTargetEye = StereoTargetEyeMask.None;
 			m_SmoothCamera.fieldOfView = m_FieldOfView;
 
-			m_Position = Vector3.Lerp(m_Position, m_VRCamera.transform.position, Time.deltaTime * m_SmoothingMultiplier);
-			m_Forward = Vector3.Lerp(m_Forward, m_VRCamera.transform.forward, Time.deltaTime * m_SmoothingMultiplier);
+			m_Position = Vector3.Lerp(m_Position, m_VRCamera.transform.localPosition, Time.deltaTime * m_SmoothingMultiplier);
+			m_Forward = Vector3.Lerp(m_Forward, m_VRCamera.transform.localRotation * Vector3.forward, Time.deltaTime * m_SmoothingMultiplier);
```

Comment:
I thought the LookRotation error was only for input vectors that are too close to zero magnitude. Is there a good way of avoiding this other edge case? Just check if the dot product is too close to 1 or the distance is too close to 0?

The reason we're using a forward direction and not the whole rotation is to eliminate roll. AFAIK, trying to pull roll out of a quaternion doesn't work in the general case without gimbal lock somewhere. I didn't write the original code, though, so I can't really say.